vmdb/product/alerts/rss/lifecycle_events.yml

```diff
@@ -29,7 +29,7 @@ orderby: "created_on DESC"
 
 # Included tables and columns for query performance
 include:
-  :vm: {}
+  :vm_or_template: {}
```

Comment:
@gmcculloug Hi Greg, the RSS feed for this was actually broken, once adding LifecycleEvent's the RSS feed would give a stack trace. Needed the above change to work.